.evergreen/auth_oidc/README.md

```diff
@@ -22,13 +22,14 @@ OIDC_ISSUER_2_URI       # The issuer URI for mock IdP 2
 
 ### Prerequisites
 
-See the 
 The `oidc_get_tokens.sh` script will automatically fetch the credentials from the `drivers/oidc` vault.
+See [Secrets Handling](../secrets_handling/README.md) for details on how the script accesses the vault.
+Add `secrets-export.sh` to your `.gitignore` to prevent checking in credentials in your repo.
 
 ### Usage
 
-Use the `oidc_get_tokens.sh` script to create a set of OIDC tokens in a temporary, including
-`test_user1` and `test_user1_expires`.  The temp file location is exported as `OIDC_TOKEN_DIR`
+Use the `oidc_get_tokens.sh` script to create a set of OIDC tokens in a temporary director, including
+`test_user1` and `test_user1_expires`.  The temp file location is exported as `OIDC_TOKEN_DIR`.
 
 ```bash
```

Comment:
unfinished ```?

Reply:
I don't follow, this block looks correct to me.

Reply:
@NoahStapp the code block is correctly closed in line 37, although this is hidden by the collapsed diff.